src/wmtk/operations/tri_mesh/EdgeCollapse.hpp

```diff
@@ -55,7 +55,7 @@ class EdgeCollapse : public TriMeshOperation, private TupleOperation
 
 private:
     Tuple m_output_tuple;
-    const OperationSettings<EdgeCollapse>& m_settings;
+    // const OperationSettings<EdgeCollapse>& m_settings;
```

Comment:
why remove the settings? i think there is a future refactor to handle how we pass these settings around but for now this practice is the "way" we're doing things?

Reply:
they were unused :)

Reply:
Please open up an issue for that but do not just comment out stuff.